src/components/tables/DataExplorerTable/index.jsx

```diff
@@ -1,17 +1,133 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import FileSaver from 'file-saver';
 import { Table } from '@arranger/components/dist/Arranger';
+import dictIcons from '../../../img/icons/index';
+import IconicButton from '../../buttons/IconicButton';
 import './DataExplorerTable.less';
 
+const DOWNLOAD_MANIFEST_BUTTON_TEXT = 'Download Manifest';
+const UUID_FIELD = 'uuid';
+const SUBJECT_ID_FIELD = 'subject_id';
```

Comment:
this should be configurable too, and we should rename it to be something more general, like 'reference_id' or something, the portal can be used for non-biomedical commons